vendor/patch/src/patch.ml

```diff
@@ -52,18 +52,31 @@ type hunk = {
 }
 
 let unified_diff hunk =
+  let left_line_num = ref hunk.their_start in 
+  let right_line_num = ref hunk.mine_start in
+  let incr ref = 
```

Comment:
This function already exists in the standard library and is called exactly like that.